faq.md

```diff
@@ -5,6 +5,27 @@ title: Frequently Asked Questions (FAQ)
 - this list will be replaced by the Table of Contents
 {:toc}
 
+## General
+
+### How do I contact the Cockpit team?
+
+- If you're filing an issue, use our issue tracker.
+   - For general Cockpit bugs and feature requests, use [Cockpit's issue tracker](https://github.com/cockpit-project/cockpit/issues).
+   - For other parts of Cockpit use specific issues:
+      - [Website issues](https://github.com/cockpit-project/cockpit-project.github.io/issues)
+      - [Virtual machine related issues](https://github.com/cockpit-project/cockpit-machines/issues)
+      - [Podman UI related issues](https://github.com/cockpit-project/cockpit-podman/issues)
+      - [OStree related issues](https://github.com/cockpit-project/cockpit-ostree/issues)
+    - If you're not sure where to file the issue, [use the general issue tracker](https://github.com/cockpit-project/cockpit/issues). (We can move it if necessary.)
+    - If it's not an issue, ask us in the forum or IRC.
+- There's [a discussion forum for Cockpit on GitHub](https://github.com/cockpit-project/cockpit/discussions).
+- The Cockpit team is on IRC, usually during European business hours, from Monday through Friday, at #cockpit on irc.libera.chat. You need [a registered nick](https://libera.chat/guides/registration) in order to join.
+- [Cockpit has a mailing list](https://lists.fedorahosted.org/admin/lists/cockpit-devel.lists.fedorahosted.org/).
+
+### How do I create an application for Cockpit?
+
+All development-focused documentation is located on [the Contribute page](https://cockpit-project.org/external/wiki/Contributing.html) of the website. This includes creating your own [app page with the starter kit](blog/cockpit-starter-kit.html) and other related topics.
```

Comment:
```suggestion
All development-focused documentation is located on the [Contributing page](https://cockpit-project.org/external/wiki/Contributing.html) of the website. This includes creating your own [app page with the starter kit](blog/cockpit-starter-kit.html) and other related topics.
```

Reply:
It's literally called "Contribute" in the navigation. I originally had it like your suggestion but realize that and changed it accordingly. But I'm fine with your suggestion too.

Reply:
Ah, I see -- the menu says "Contribute", the page title "Contributing". Not important enough. Including "the" in the link feels a bit strange, but again,  not important.